Cargo.toml

```diff
@@ -534,6 +534,9 @@ libm = ["bevy_internal/libm"]
 # Enables use of browser APIs. Note this is currently only applicable on `wasm32` architectures.
 web = ["bevy_internal/web"]
 
+# Enable hotpatching of Bevy systems
+hotpatching = ["bevy_internal/hotpatching"]
```

Comment:
Since we always write "hot-reloading" with a dash, it would probably be nice to do the same with "hot-patching". Note that Dioxus itself is inconsistent with this :D

Reply:
features are named with `_` rather than `-`, and I find `hot_patching` unpleasant to read...